<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LOGIN </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: qlineargradient(spread:pad, x1:0.517, y1:0, x2:0.522, y2:1, stop:0 rgba(85, 0, 127, 255), stop:1 rgba(159, 159, 159, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
    </property>
    <property name="text">
     <string>DATADIMENSION</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: rgba(64, 64, 64, 0.5); /* Fondo plomo suave pero más oscuro del QLabel */
    border: 2px solid #00aaff; /* Borde azul del QLabel */
    border-radius: 5px; /* Borde redondeado */
    padding: 2px; /* Espaciado interno */
    font-size: 21px; /* Tamaño de fuente aumentado */
    color: #ffffff; /* Color del texto en blanco */
    text-align: center; /* Texto centrado */
}

QLabel:hover {
    background-color: rgba(64, 64, 64, 0.7); /* Fondo del QLabel al pasar el mouse */
}

QLabel::selection {
    background-color: #00aaff; /* Fondo del QLabel al seleccionar el texto */
    color: #ffffff; /* Color del texto al seleccionar el texto */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Usuario:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>290</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    background-color: rgba(64, 64, 64, 0.5); /* Fondo plomo suave pero más oscuro del QLabel */
    border: 2px solid #00aaff; /* Borde azul del QLabel */
    border-radius: 5px; /* Borde redondeado */
    padding: 2px; /* Espaciado interno */
    font-size: 21px; /* Tamaño de fuente aumentado */
    color: #ffffff; /* Color del texto en blanco */
    text-align: center; /* Texto centrado */
}

QLabel:hover {
    background-color: rgba(64, 64, 64, 0.7); /* Fondo del QLabel al pasar el mouse */
}

QLabel::selection {
    background-color: #00aaff; /* Fondo del QLabel al seleccionar el texto */
    color: #ffffff; /* Color del texto al seleccionar el texto */
}</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Contraseña:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #f7f7f7; /* Fondo del QLineEdit */
    border: 1px solid #d0d0d0; /* Borde del QLineEdit */
    border-radius: 5px; /* Borde redondeado */
    padding: 2px; /* Espaciado interno */
    font-size: 18px; /* Tamaño de fuente */
    color: #333333; /* Color del texto */
}

QLineEdit:hover {
    background-color: #ebebeb; /* Fondo del QLineEdit al pasar el mouse */
}

QLineEdit:focus {
    background-color: #ffffff; /* Fondo del QLineEdit al tener el foco */
    border: 1px solid #4da8da; /* Borde del QLineEdit al tener el foco */
}

QLineEdit::placeholder {
    color: #999999; /* Color del texto de marcador de posición */
}

QLineEdit::hover::placeholder,
QLineEdit::focus::placeholder {
    color: #666666; /* Color del texto de marcador de posición al pasar el mouse o tener el foco */
}

QLineEdit::hover::focus {
    background-color: #ffffff; /* Fondo del QLineEdit al tener el foco y pasar el mouse */
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: #f7f7f7; /* Fondo del QLineEdit */
    border: 1px solid #d0d0d0; /* Borde del QLineEdit */
    border-radius: 5px; /* Borde redondeado */
    padding: 2px; /* Espaciado interno */
    font-size: 18px; /* Tamaño de fuente */
    color: #333333; /* Color del texto */
}

QLineEdit:hover {
    background-color: #ebebeb; /* Fondo del QLineEdit al pasar el mouse */
}

QLineEdit:focus {
    background-color: #ffffff; /* Fondo del QLineEdit al tener el foco */
    border: 1px solid #4da8da; /* Borde del QLineEdit al tener el foco */
}

QLineEdit::placeholder {
    color: #999999; /* Color del texto de marcador de posición */
}

QLineEdit::hover::placeholder,
QLineEdit::focus::placeholder {
    color: #666666; /* Color del texto de marcador de posición al pasar el mouse o tener el foco */
}

QLineEdit::hover::focus {
    background-color: #ffffff; /* Fondo del QLineEdit al tener el foco y pasar el mouse */
}
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #4CAF50, stop: 1 #2E7D32);
    border: 2px solid #2E7D32;
    border-radius: 5px;
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 5px 20px;
    box-shadow: 2px 2px 2px rgba(0,0,0,0.3);
}
QPushButton:hover {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2E7D32, stop: 1 #4CAF50);
    cursor: pointer;
}
QPushButton:pressed {
    background-color: #2E7D32;
    border: 2px solid #4CAF50;
}
</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #FF5722, stop: 1 #E64A19);
    border: 2px solid #E64A19;
    border-radius: 5px;
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 5px 20px;
    box-shadow: 2px 2px 2px rgba(0,0,0,0.3);
}
QPushButton:hover {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #E64A19, stop: 1 #FF5722);
    cursor: pointer;
}
QPushButton:pressed {
    background-color: #E64A19;
    border: 2px solid #FF5722;
}

</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(252, 255, 39);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-30</y>
      <width>721</width>
      <height>541</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/Login_principal.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2196F3, stop: 1 #0D47A1);
    border: 2px solid #0D47A1;
    border-radius: 5px;
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 5px 20px;
    box-shadow: 2px 2px 2px rgba(0,0,0,0.3);
}
QPushButton:hover {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #0D47A1, stop: 1 #2196F3);
    cursor: pointer;
}
QPushButton:pressed {
    background-color: #0D47A1;
    border: 2px solid #2196F3;
}

</string>
    </property>
    <property name="text">
     <string>Registrarse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/Login.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_6</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>lineEdit_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_5</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
